metron-analytics/metron-profiler/README.md

```diff
@@ -204,21 +204,21 @@ The Profiler runs as an independent Storm topology.  The configuration for the P
-| profiler.period.duration | The duration of each profile period.  This value should be defined along with `profiler.period.duration.units`.  |
-| profiler.period.duration.units | The units used to specify the `profiler.period.duration`. |
-| profiler.ttl | If a message has not been applied to a Profile in this period of time, the Profile will be forgotten and its resources will be cleaned up. This value should be defined along with `profiler.ttl.units`. |
-| profiler.ttl.units | The units used to specify the `profiler.ttl`. |
-| profiler.hbase.salt.divisor  |  A salt is prepended to the row key to help prevent hotspotting.  This constant is used to generate the salt.  Ideally, this constant should be roughly equal to the number of nodes in the Hbase cluster.  |
-| profiler.hbase.table | The name of the HBase table that profiles are written to.  |
-| profiler.hbase.column.family | The column family used to store profiles. |
-| profiler.hbase.batch | The number of puts that are written in a single batch.  |
-| profiler.hbase.flush.interval.seconds | The maximum number of seconds between batch writes to HBase. |
+| Setting                               | Description
+|---                                    |---
+| profiler.workers                      | The number of worker processes to create for the topology.
+| profiler.executors                    | The number of executors to spawn per component.
+| profiler.input.topic                  | The name of the Kafka topic from which to consume data.
+| profiler.output.topic                 | The name of the Kafka topic to which profile data is written.  Only used with profiles that use the [`triage` result field](#result).
+| profiler.period.duration              | The duration of each profile period.  This value should be defined along with `profiler.period.duration.units`.
+| profiler.period.duration.units        | The units used to specify the `profiler.period.duration`.
+| profiler.ttl                          | If a message has not been applied to a Profile in this period of time, the Profile will be forgotten and its resources will be cleaned up. This value should be defined along with `profiler.ttl.units`.
+| profiler.ttl.units                    | The units used to specify the `profiler.ttl`.
+| profiler.hbase.salt.divisor           | A salt is prepended to the row key to help prevent hotspotting.  This constant is used to generate the salt.  Ideally, this constant should be roughly equal to the number of nodes in the Hbase cluster.
+| profiler.hbase.table                  | The name of the HBase table that profiles are written to.
+| profiler.hbase.column.family          | The column family used to store profiles.
+| profiler.hbase.batch                  | The number of puts that are written in a single batch.
+| profiler.hbase.flush.interval.seconds | The maximum number of seconds between batch writes to HBase.
```

Comment:
That link you provided says:
> The pipes on either end of the table are optional.

which makes me believe that either format would be considered correct.  

That said, I removed them here because I previously [made a similar change](https://github.com/apache/incubator-metron/commit/f6c253f44fe7ca5a6b31c46075f537940751c934#diff-1e173a5423bfe7e7b564802de834ae54) to square up another table in the same README and add terminal "|"s, however it was [promptly reverted by a later commit](https://github.com/apache/incubator-metron/commit/818b0b17b131d875259178e61e59d32d02ae792a#diff-1e173a5423bfe7e7b564802de834ae54).  My goal with the table updates is simply to:
1.  Format tables somewhat consistently across the documentation.
1.  Have tables parse properly with doxia and github md.
1.  Make tables easy to edit and review in a text editor to avoid issues like [this](https://github.com/apache/incubator-metron/commit/f6c253f44fe7ca5a6b31c46075f537940751c934#diff-dba284384333126318019f762880ace4) (line 39), where there was accidentally a fourth column that wasn't being properly displayed.

I would be more than happy to reinstate the terminal "|"s.  As long as the previous goals are accomplished, I don't care which format we settle on, as long as we set a standard that we can stick to and don't keep changing things back and forth.